zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/exception/NotificatiesException.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.notificatiesapi.exception
+
+import org.springframework.http.HttpStatus
+
+class NotificatiesException(message: String?, val status: HttpStatus) : RuntimeException(message) {
```

Comment:
This exception doesn't really say a whole lot. Generally speaking, exceptions say something about the kind of exception that is being thrown. In this case, it mentions the domain the error is related to.

Reply:
well this was only to wrap some exception that may be thrown by the listeners. each listener may have different reasons for throwing exceptions. if they want it to be captured and sent out they should extend this exception or throw it directly.